guidance.md

```diff
@@ -1 +1,139 @@
+
+> TODO: Insert specific examples or verbatims here
+
+### Issue 3 - Client library specific issues with the earlier azure API versioning guidance
+
+The previous official versioning guidelines state that each new API version corresponds to a new major version of a client library. This has several challenges:
+
+- Many languages/runtimes are unable to load multiple versions of the same library in the same process.
+- New major library versions cause great disruption in those ecosystems and should be avoided. One of the most common issues is the diamond reference problem where all libraries have to agree on the one and only version they reference.
+- For language ecosystems where semantic versioning is used for library versioning, the bar for what a semantic major version is differs from the bar for "changes that require a service api version update". Many "changes that require a service api version update" are minor version changes according to semantic versioning. Strictly enforcing new major version of client library per API version would introduce unnescessary major version.
+- An alternative to versioning packages is to create packages with the version number in the package name (e.g. instead of having `Azure.Storage` v17 followed by `Azure.Storage` v18, you have `Azure.StorageV17` v1 followed by `Azure.StorageV18` v1). However, this causes an explosion of the number of packages. This impacts how easy it is to find the correct package for a given service (the package manager's search engine generally boost the most commonly downloaded package, which means that new versions are effectively hidden from search).
+It also prevents tooling from helping you upgrade versions since tools don't understand that the different packages can be treated as different versions of the same package. 
+
+As an example of how many major versions of a given package we are currently releasing, [azure-mgmt-storage](https://pypi.org/project/azure-mgmt-storage/8.0.0/) for Python has had 5 major version changes since 2019.
+
+## Proposed changes and additions
+
+The proposed changes below are in addition to providing more tooling to both help review and detect changes as they are deployed. It is fully understood that guidance alone does not solve the problem.
+
+### Proposal 1 - Updated SDK versioning proposal
+
+In order to address [Issue 3](#-Issue-3---Client-library-specific-issues-with-the-earlier-azure-API-versioning-guidance) listed above, the updated Azure SDK client library proposal is to, instead of introducing a new major library version for a new API version, *add support for multiple service api versions in a single library*. By default, the latest service API version at the point of the release of the library is used, but a developer can opt in to earlier API versions.
```

Comment:
By opting into an unsupported version, the then SLA will change right?  Because we can't guarantee the SDK works against that version because we are testing against it.